gammapy/spectrum/tests/test_spectrum_dataset.py

```diff
@@ -0,0 +1,138 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import pytest
+from numpy.testing import assert_allclose
+import astropy.units as u
+import numpy as np
+from ...utils.testing import requires_data, requires_dependency
+from ...utils.random import get_random_state
+from ...irf import EffectiveAreaTable, EnergyDispersion
+from ...utils.fitting import Fit
+from ..models import PowerLaw, ConstantModel
+from ...spectrum import (
+    PHACountsSpectrum,
+    ONOFFSpectrumDataset
+)
+
+
+
+class Test_ONOFFSpectrumDataset:
```

Comment:
Remove the underscore...class names are recommended to use camel-case in PEP8.

Reply:
OK